lib/elastic_apm/spies/mongo.rb

```diff
@@ -41,9 +41,15 @@ def push_event(event)
           # Some MongoDB commands are not on collections but rather are db
           # admin commands. For these commands, the value at the `command_name`
           # key is the integer 1.
-          unless event.command[event.command_name] == 1
-            collection = event.command[event.command_name]
-          end
+          # For getMore commands, the value at `command_name` is the cursor id
+          # and the collection name is at the key `collection`
+          collection = if event.command[event.command_name] == 1 ||
```

Comment:
```ruby
collection =
  if event.command[event.command_name] == 1 ||
    event.command[event.command_name].is_a?(BSON::Int64)
    event.command[:collection]
  else
    event.command[event.command_name]
  end
```

Not sure if Rubocop will complain but fwiw I hate arbitrary indenting levels. If you break it like above instead, it's just 2 spaces like everywhere else.

💅, I know.

Reply:
Sure, I'll try that and see if rubocop complains.